src/test/java/teammates/test/cases/storage/FeedbackSessionsDbTest.java

```diff
@@ -211,6 +211,32 @@ private void testGetSoftDeletedFeedbackSessionsForCourse() {
         assertTrue(fsDb.getSoftDeletedFeedbackSessionsForCourse("idOfCourseNoEvals").isEmpty());
     }
 
+    @Test
+    public void testSoftDeleteFeedbackSession() throws Exception {
+        FeedbackSessionAttributes fs = getNewFeedbackSession();
+        fsDb.createEntity(fs);
+
+        ______TS("Success: soft delete an existing feedback session");
+        fsDb.softDeleteFeedbackSession(fs.getFeedbackSessionName(), fs.getCourseId());
+
+        assertNull(fsDb.getFeedbackSession(fs.getFeedbackSessionName(), fs.getCourseId()));
+        assertNotNull(fsDb.getSoftDeletedFeedbackSession(fs.getCourseId(),
+                fs.getFeedbackSessionName()));
+
+        ______TS("Success: restore soft deleted course");
+        fsDb.restoreDeletedFeedbackSession(fs.getFeedbackSessionName(), fs.getCourseId());
+
+        assertNull(fsDb.getSoftDeletedFeedbackSession(fs.getCourseId(),
```

Comment:
assertNotNull(fsDb.getFeedbackSession(fs.getFeedbackSessionName(), fs.getCourseId()))

Reply:
assertFalse(fsDb.getFeedbackSession(fs.getCourseId(), fs.getFeedbackSessionName()).isSessionDeleted());

I have this below though